src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -46,28 +46,25 @@ const styles = StyleSheet.create({
     backgroundColor: COLORS.anon_primary
   },
   textContainer: {
-    paddingLeft: BUBBLE_LEFT_PADDING,
+    paddingLeft: BUBBLE_LEFT_PADDING + BUBBLE_LEFT_PADDING,
     paddingRight: BUBBLE_RIGHT_PADDING,
     paddingTop: 4,
     paddingBottom: 4,
-    borderRadius: 8,
+    borderRadius: 12,
     borderTopEndRadius: 0,
     flex: 1
   },
-  textContainerNewLine: {
-    paddingBottom: 14
-  },
+  textContainerNewLine: {},
   userText: {
     fontFamily: fonts.inter[600],
     fontSize: 12,
-    lineHeight: 19.36,
+    lineHeight: 18,
     color: COLORS.white
   },
   text: {
     fontFamily: fonts.inter[400],
     fontSize: 16,
-    lineHeight: 19.36,
-    marginBottom: 4,
+    lineHeight: 24,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The lineHeight for text has been increased significantly from 19.36 to 24, and marginBottom has been removed. This could impact the spacing between lines and elements. Please verify these changes against your design specifications.
```diff
- lineHeight: 24,
+ lineHeight: 19.36,
+ marginBottom: 4,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->